src/components/OSCALControl.test.js

```diff
@@ -1,10 +1,34 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render, screen, fireEvent } from "@testing-library/react";
 import OSCALControl from "./OSCALControl";
+import OSCALControlImplementation from "./OSCALControlImplementation";
+import { controlImplTestData, exampleControl } from "../test-data/ControlsData";
+import { exampleComponents } from "../test-data/ComponentsData";
+import { profileModifySmtTestData } from "../test-data/ModificationsData";
 
 test("OSCALCatalog displays controls", () => {
   const testControl = { id: "ac-1" };
   render(<OSCALControl control={testControl} />);
   const result = screen.getByText("ac-1");
   expect(result).toBeVisible();
 });
+
+test("OSCALControl displays _smt modifications", async () => {
```

Comment:
```suggestion
test("OSCALControl displays statement modifications", async () => {
```

Just one nitpick - this can be a tad more descriptive